io.catenax.shared.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,144 @@
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.message_header:1.0.0#> .
+
+:MessageHeaderAspect a samm:Aspect ;
+   samm:preferredName "Message Header Aspect"@en ;
+   samm:description "Aspect model describing the shared message header."@en ;
+   samm:properties ( :header ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:header a samm:Property ;
+   samm:preferredName "Header"@en ;
+   samm:description "Contains standardized attributes for message processing common across several use cases."@en ;
+   samm:characteristic :HeaderCharacteristic .
+
+:HeaderCharacteristic a samm:Characteristic ;
+   samm:preferredName "Header Characteristic"@en ;
+   samm:description "Characteristic describing the common shared aspect Message Header"@en ;
+   samm:dataType :MessageHeader .
+
+:MessageHeader a samm:Entity ;
+   samm:preferredName "Message Header"@en ;
+   samm:description "The Catena-X Message Header contains standardized attributes for message processing that are common across several use cases. It targets a minimal set of aspects that are needed for overall message handling, and not for the processing of the business related payload part. By standardizing the message header structure, we support a harmonized modelling approach and re-use of concepts. The message header originated from discussions on push notifications but has since then been abstracted for overall message handling – and does not only apply to notification anymore."@en ;
+   samm:properties ( :messageId :context :sentDateTime :senderBpn :receiverBpn [ samm:property :expectedResponseBy; samm:optional true ] [ samm:property :relatedMessageId; samm:optional true ] :version ) .
+
+:messageId a samm:Property ;
+   samm:preferredName "Message ID"@en ;
+   samm:description "Unique ID identifying the message. The purpose of the ID is to uniquely identify a single message, therefore it MUST not be reused."@en ;
+   samm:characteristic :UuidCharacteristic ;
+   samm:exampleValue "3b4edc05-e214-47a1-b0c2-1d831cdd9ba9" .
+
+:context a samm:Property ;
+   samm:preferredName "Context"@en ;
+   samm:description """Information about the context the message should be considered in.
+Default: For Use Cases which need to validate against the Semantic Hub automatically.
+Fallback: For Use Cases which do not use automatic validation.
+In both cases: Versioning only refers to major versions.
+String pattern of the value follows: <domain>-<subdomain>-<object>-<major version>"""@en ;
+   samm:characteristic :ContextCharacteristic ;
+   samm:exampleValue "urn:samm:io.catenax.<trace-QM-Alert>:1.x.x" .
```

Comment:
Proposal for an unambiguous and use case / domain neutral example value consistent with the proposed description update:

`urn:samm:io.catenax.<ASPECT-MODEL-NAME>:1.x.x`